src/sync/app.hpp

```diff
@@ -246,31 +192,32 @@ class App {
         App* parent;
     };
 
-    /**
-    Log in a user and asynchronously retrieve a user object.
-
-    If the log in completes successfully, the completion block will be called, and a
-    `SyncUser` representing the logged-in user will be passed to it. This user object
-    can be used to open `Realm`s and retrieve `SyncSession`s. Otherwise, the
-    completion block will be called with an error.
-
-    - parameter credentials: A `SyncCredentials` object representing the user to log in.
-    - parameter completion: A callback block to be invoked once the log in completes.
-    */
+    /// Log in a user and asynchronously retrieve a user object.
+    /// If the log in completes successfully, the completion block will be called, and a
+    /// `SyncUser` representing the logged-in user will be passed to it. This user object
+    /// can be used to open `Realm`s and retrieve `SyncSession`s. Otherwise, the
+    /// completion block will be called with an error.
+    ///
+    /// @param credentials A `SyncCredentials` object representing the user to log in.
+    /// @param completion_block A callback block to be invoked once the log in completes.
     void log_in_with_credentials(const AppCredentials& credentials,
                                  std::function<void(std::shared_ptr<SyncUser>, Optional<AppError>)> completion_block) const;
 
-    /**
-     Logout the current user.
-     */
+    /// Logout the current user.
     void log_out(std::function<void(Optional<AppError>)>) const;
 
-    // Get a provider client for the given class type.
+    /// Get a provider client for the given class type.
     template <class T>
     T provider_client() {
         return T(this);
     }
-
+    
+    /// Retrieves a general-purpose service client for the Stitch service
+    /// associated with the specified name. Use this for services which do not
+    /// have a well-defined interface in the SDK.
+    /// @param service_name  The name of the desired service in MongoDB Stitch.
+    RemoteMongoClient service_client(std::string service_name);
```

Comment:
[minor] Should be `RemoteMongoClient remote_mongo_client()`

Reply:
Addressed